src/Field/BasicField.php

```diff
@@ -115,7 +115,7 @@ public function fetchFieldKey($fieldName)
      */
     public function fetchFieldType($fieldKey)
     {
-        $post = $this->post->orWhere(function ($query) use ($fieldKey) {
+        $post = Post::where(function ($query) use ($fieldKey) {
```

Comment:
Hi, @njbarrett thanks for the contribution. I guess changing this to `Post::where()` we'll lose the database connection. `$this->post` comes with the connection set by the user, so it has the proper connection name. Once change this we can break the code.

I think there's no test for database connection yet, but it's something I'd like to write (or you can of course) and make sure the same connection is being used for the `acf` plugin as well :-)